src/components/InfoPanel.component.js

```diff
@@ -1,8 +1,42 @@
 import React, { Component } from 'react'
+import { Image } from 'react-bootstrap'
+import './../styles/App.css'
 
 class InfoPanel extends Component {
+  constructor(props) {
+    super(props)
+  }
+
   render() {
-    return <div>InfoPanel</div>
+    const selectedEvent = this.props.selectedEvent
+    const img_url = selectedEvent.image
+
+    return (
+      <div className="info-panel">
+        <h1>
+          <u>
```

Comment:
It's not the best practice to use html tags for styling. Styling should be done with css. For now, we don't have to worry about styling too much yet, but let's not use html styling tags.